project/Dependencies.scala

```diff
@@ -33,6 +33,7 @@ object Dependencies {
   val ficus: ModuleID =          "com.iheart"                 %% "ficus"           % "1.4.0"
   val cats: ModuleID =           "org.typelevel"              %% "cats"            % "0.9.0"
   val httpClient: ModuleID =     "org.apache.httpcomponents"  % "httpclient"       % "4.5.3"  // upgrading a transitive dependency to avoid security warnings
+  val enumeratum: ModuleID =     "com.beachape"               %% "enumeratum"      % "1.5.12"
```

Comment:
Consider this a proposal that we standardize on [enumeratum](https://github.com/lloydmeta/enumeratum) for all our enum-ing needs.